presentation/src/main/java/com/example/presentation/ui/detail/DetailScreen.kt

```diff
@@ -0,0 +1,208 @@
+package com.example.presentation.ui.detail
+
+import android.util.Log
+import androidx.annotation.StringRes
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.presentation.R
+import com.example.presentation.model.UserInfo
+import com.example.presentation.ui.components.GradientProfileImage
+import com.example.presentation.ui.components.GradientTextBox
+import com.example.presentation.ui.components.HorizontalDivider
+import com.example.presentation.ui.home.HomeViewModel
+import com.example.presentation.ui.theme.GitHubLink
+import model.User
+
+@Composable
+fun DetailRoute(
+    modifier: Modifier = Modifier,
+    viewModel: HomeViewModel,
+    index: Int,
+) {
+    val userResults by viewModel.userResults.collectAsState()
```

Comment:
```suggestion
    val userResults by viewModel.userResults.collectAsState()
```

저번에 다연이가 포스팅해준 compose에서 flow 수집하는 방법 읽어서 적용해봐도 좋을 것 같아요 :)
https://reco-dy.tistory.com/13